util/topsql/stmtstats/stmtstatstest/stmtstats_test.go

```diff
@@ -118,6 +118,7 @@ func TestExecCount(t *testing.T) {
 			if _, ok := sqlDigests[digest.SQLDigest]; ok {
 				found++
 				assert.Equal(t, uint64(ExecCountPerSQL), item.ExecCount)
+				assert.True(t, item.SumExecNanoDuration > uint64(time.Nanosecond*ExecCountPerSQL))
```

Comment:
This lower bound is not strong enough. time.Nanosecond is simply impossible to achieve, which means even if you count the time incorrectly this test will pass.

Reply:
An idea: the upper bound can be obtained by calculating the duration of `tk.MustExec()` (take the max duration in multiple executions), and the lower bound can be set as the sum of some exact partial execution duration such as `parsing time` (how to obtain these times is a problem, which may require adding a lot of code).

Reply:
resolved, PTAL